ci/datadog-synthetics.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
It would be good to add some comment which tells in short what the workflow does. For example, see [here](https://github.com/actions/starter-workflows/blob/51e7c8e1e6c69cb6b41e142d579da95a44191ce3/ci/python-publish.yml#L1)

Reply:
May also consider to add a doc link where user can find more details